pkg/machines/hostvmslist.jsx

```diff
@@ -104,6 +106,26 @@ class HostVmsList extends React.Component {
                         onChange={this.onSearchInputChange}
                         placeholder={_("Filter by name")} />
                 </ToolbarItem>
+                <ToolbarItem variant="label" id="vm-state-select">
+                    {_("State")}
+                </ToolbarItem>
+                <ToolbarItem>
+                    <Select variant={SelectVariant.single}
+                            toggleId="vm-state-select-toggle"
+                            onToggle={statusIsExpanded => this.setState({ statusIsExpanded })}
+                            onSelect={(event, selection) => this.setState({ statusIsExpanded: false, statusSelected: selection })}
+                            selections={this.state.statusSelected}
+                            isOpen={this.state.statusIsExpanded}
+                            aria-labelledby="vm-state-select">
+                        {[
+                            { value: _("All"), },
+                            { value: _("Running"), apiState: "running" },
+                            { value: _("Shut off"), apiState: "shut off" }
```

Comment:
Do you intend to add the other states like "paused" as well?

Reply:
I don't think that's necessary, filtering my the major states should be fine.
(We do the similar thing in podman page - filter by running or show everything)

Reply:
OK. I thought about filtering for "error" states like pause might come in handy, but we can add that later if requested.